<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pw_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>305</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;HY견명조&quot;;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="how_to_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>95</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>사용 방법</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_num_btn">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>220</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>인증 번호 생성</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>260</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>아래 버튼을 눌러 인증번호를 생성하세요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>generate_num_btn</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>how_to_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>generate_num_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>generate_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>how_to_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>how_to()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>start()</signal>
  <slot>generate_num()</slot>
  <slot>login()</slot>
  <slot>how_to()</slot>
 </slots>
</ui>
